libraries/ESP8266WiFi/src/include/UdpContext.h

```diff
@@ -483,7 +516,11 @@ class UdpContext
 #ifdef LWIP_MAYBE_XCC
     uint16_t _mcast_ttl;
 #endif
-    IPAddress _src_addr, _dst_addr;
+    struct addrhelper_s
```

Comment:
Consider separating the definition from the declaration:
```cpp
struct addrHelper
{
...
};
addrHelper _current_addr;
```

Also, notice camelCase and underscore prefix.